src/librawspeed/decoders/MrwDecoder.h

```diff
@@ -31,10 +31,10 @@ namespace RawSpeed {
 class CameraMetaData;
 class TiffIFD;
 
-typedef struct {
+using mrw_camera_t = struct mrw_camera_t {
```

Comment:
No. `struct` and `class` behave exactly the same except that all members of a `struct` are `public` by default, instead of `private`.